app/uk/gov/hmrc/incometaxpenaltiesfrontend/views/helpers/LSPCardHelper.scala

```diff
@@ -141,7 +144,7 @@ class LSPCardHelper @Inject()(summaryRow: LSPSummaryListRowHelper) extends Summa
     )
   }
 
-  def removedPointCard(penalty: LSPDetails, pointsRemovedAfterPeriodOfCompliance: Boolean)(implicit messages: Messages): LateSubmissionPenaltySummaryCard = {
+  def removedPointCard(penalty: LSPDetails, pointsRemovedAfterPeriodOfCompliance: Boolean)(implicit messages: Messages, timeMachine: TimeMachine): LateSubmissionPenaltySummaryCard = {
```

Comment:
you don't need them in these functions as they are implicitly passed into the class